### PR TITLE
Add csi-driver-manila-operator image

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

**Comment by Fedosin** [comment]
@pierreprinetti We run just go-fmt.sh for testing at this moment, and we don't need boscos for this. Later, after integration with cluster-storage-operator, there will be e2e-openstack tests, and boscos will be necessary.

File: core-services/image-mirroring/openshift/mapping_origin_4_5 [hunk 1]
```diff
@@ -65,6 +65,7 @@ registry.svc.ci.openshift.org/origin/4.5:coredns quay.io/openshift/origin-coredn
 registry.svc.ci.openshift.org/origin/4.5:crd-schema-gen quay.io/openshift/origin-crd-schema-gen:4.5 quay.io/openshift/origin-crd-schema-gen:4.5.0 quay.io/openshift/origin-crd-schema-gen:latest
 registry.svc.ci.openshift.org/origin/4.5:csi-cluster-driver-registrar quay.io/openshift/origin-csi-cluster-driver-registrar:4.5 quay.io/openshift/origin-csi-cluster-driver-registrar:4.5.0 quay.io/openshift/origin-csi-cluster-driver-registrar:latest
 registry.svc.ci.openshift.org/origin/4.5:csi-driver-manila quay.io/openshift/origin-csi-driver-manila:4.5 quay.io/openshift/origin-csi-driver-manila:4.5.0 quay.io/openshift/origin-csi-driver-manila:latest
+registry.svc.ci.openshift.org/origin/4.5:csi-driver-manila-operator quay.io/openshift/origin-csi-driver-manila-operator:4.5 quay.io/openshift/origin-csi-driver-manila-operator:4.5.0
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_5]: You should not be adding these until the images are built at least once.

Comment (Fedosin) [reply core-services/image-mirroring/openshift/mapping_origin_4_5]: done!

**Pull request merged**